table/src/main/java/com/bazaarvoice/emodb/table/db/astyanax/AstyanaxTableDAO.java

```diff
@@ -506,7 +537,20 @@ private void updateTableMetadata(String table, Delta delta, Audit audit, @Nullab
         }
     }
 
-    @Timed(name = "bv.emodb.table.AstyanaxTableDAO.create", absolute = true)
+    /* Write the delta to the "metadata changes" system table and invalidate caches in the specified scope.
+     *  "metadata changes" system table stores all the drop, purge, attribute changes, placement moves etc information of the tables.
+     */
+    private void writeTableMetadataChanges(String key, Delta delta, Audit audit, @Nullable InvalidationScope scope) {
+        _backingStore.update(_systemTableMetadataChanges, key, TimeUUIDs.newUUID(), delta, audit,
+                scope == InvalidationScope.GLOBAL ? WriteConsistency.GLOBAL : WriteConsistency.STRONG);
+
+        // Synchronously notify other emodb servers of the table change.
+        if (scope != null) {
+            _tableCacheHandle.invalidate(scope, key);
```

Comment:
Unnecessary.  Adding new metadata doesn't need to invalidate the table itself, that'll happen in `updateTableMetadata`.